server/plugins/Slack.ts

```diff
@@ -40,4 +129,4 @@ const SlackPlugin: Plugin<{}> = {
 	init: async () => new Slack()
```

Comment:
I can't comment on it but I think the schema above should have `channels: [String!]!` to force you to but a list there. (Notice the extra `!`).